app/routes.py

```diff
@@ -1 +1,115 @@
-from flask import Blueprint
\ No newline at end of file
+from os import abort
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import date
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+#Validate
+def validate_task(task_id):
+    try:
+        task_id = int(task_id)
+    except:
+        abort(make_response({"message" : f"Task ID '{task_id}' invalid"}, 400))
+    
+    task = Task.query.get(task_id)
+
+    if not task:
+        abort(make_response({"message" : f"Task ID '{task_id}' not found"}, 404))
+    
+    return task
+
+#CREATE a task
+@tasks_bp.route("", methods=['POST'])
+def create_task():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "description" not in request_body:
+        return jsonify({"details": "Invalid data"}), 400
+
+    new_task = Task(title=request_body["title"], 
+                description=request_body["description"])
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    return jsonify({"task": new_task.to_dict()}), 201
+
+#READ all tasks
+@tasks_bp.route("", methods=['GET'])
+def get_all_tasks():
+
+    sort_query = request.args.get("sort")
+
+    if sort_query == "asc":
+        tasks = Task.query.order_by(Task.title.asc())
+    elif sort_query == "desc":
+        tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+
+    task_response = []
+    for task in tasks:
+        task_response.append(task.to_dict())
+
+    return jsonify(task_response)
```

Comment:
Even though Flask will add the 200 for you, I recommend being explicit about the code everywhere, and not just in non 200 returns.  Consistent formats make code easier to read.
```suggestion
    return jsonify(task_response), 200
```